provision/vvv-init.sh

```diff
@@ -61,3 +61,22 @@ else
     sed -i "s#{{TLS_CERT}}##" "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf"
     sed -i "s#{{TLS_KEY}}##" "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf"
 fi
+
+DELETE_DEFAULT_PLUGINS=`get_config_value 'delete_default_plugins' ''`
+if [ ! -z "${DELETE_DEFAULT_PLUGINS}" ]; then
+    noroot wp plugin delete akismet
```

Comment:
This looks ok, but we might want to document what happens if you specify to delete the default plugins, but then you put akismet and hello dolly in the list of plugins to install

Reply:
good question, they will be probably removed on next provision.
We should check if it is a new install or not to execute it but is not simple at all.

Reply:
I hadn't realised that would happen actually, how about we swap the two features around in the code so that the default plugin deletion happens second not first?

Reply:
That way it'd always give the same result